Sources/Extensions/UIKit/UITableViewExtensions.swift

```diff
@@ -65,7 +65,7 @@ public extension UITableView {
 	
 	/// SwifterSwift: Remove TableFooterView.
 	public func removeTableFooterView() {
-		tableFooterView = nil
+		tableFooterView = UIView()
```

Comment:
I'm curious why this doesn't work? 🤔 We had a passing unit test for it

Reply:
Unless I should create a separate extension for this. 
Mainly is that. Setting tablefooterview to nil you still see your separators etc. To see an actual removed tablefooterview you need to pass an empty UIView()

Reply:
@happiehappie Thanks for explaining. I did not know that. So IMO this is working properly, we just need to decide if we want to change this behavior. cc @LucianoPAlmeida

Reply:
Yea so depends on you guys, maybe you guys indeed prefer remove to be “just-back-to-default” instead of remove. I can write a different function if you guys prefer the former

Reply:
@happiehappie 
Sorry for the late response 😅
I agree with @SD10, this extension is doing what it should do by the name `removeTableFooterView`. Maybe to this separator thing is better to have another extension. But I'm not sure since it's a very simple one line of code trick like assign to a property.

Reply:
I see, it's fine, can close/delete this then :3

Reply:
The problem is this extension is also 1 line @LucianoPAlmeida. Kind of pointless 🤔

Reply:
@SD10 Yep, I feel the same but since this one is already there :) 
I think there's no problem